### PR TITLE
Fixed cluttered room error

### PR DESCRIPTION
**Pull request by adamgoerdt**

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -390,7 +390,13 @@ class Bescort
       path.each do |movement|
         move(movement)
         waitrt
-        fput('collect rock')
+        case bput('collect rock','You manage to collect a pile', 'The room is too cluttered')
```
Comment (rpherbig) [inline bescort.lic]: What would you think about moving the collect logic into `common.lic`, similar to how `forage?` is done? I see some overlap here with the collect logic in `crossing-training`.

Comment (adamgoerdt) [inline bescort.lic]: Logic has been moved and collecting in crossing-training and bescort consolidated to use it

File: common.lic [hunk 2]
```diff
@@ -93,6 +93,16 @@ module DRC
     end
     true
   end
+  
+  def collect(item)
+    case bput("collect #{item}",'You manage to collect a pile', 'The room is too cluttered')
+    when 'The room is too cluttered'
+      fput('kick pile')
```
Comment (rpherbig) [inline common.lic]: What do you think about `fput('dump junk')` after kicking?

File: common.lic [hunk 3]
```diff
@@ -93,6 +93,16 @@ module DRC
...
+      fput('kick pile')
+      fput('kick pile')
+      fput("collect #{item}")
+    end
```
Comment (rpherbig) [inline common.lic]: I recommend `waitrt?` at the end of the method (so you don't have to do it every time you call it).

File: common.lic (hunk 2 above)
Comment (adamgoerdt) [inline common.lic]: I'm thinking to err on the side of caution regarding dump junk.  We can't guarantee that there aren't people in the room who have things on the ground they don't want to lose.

File: common.lic (hunk 3 above)
Comment (adamgoerdt) [inline common.lic]: Added the wait

**Pull request merged**